documentation/pdb_client_tools.markdown

```diff
@@ -48,11 +55,49 @@ certificate-whitelist and specify the paths to the CLI node's cacert, cert, and
 private key when using the CLI either with flags or a configuration file.
 
 To configure the PuppetDB CLI to talk to your PuppetDB with flags, add a
-configuration file at `$HOME/.puppetlabs/client-tools/puppetdb.conf`. For more
+configuration file at `$HOME/.puppetlabs/client-tools/puppetdb.conf` (or
+`$USERPROFILE/.puppetlabs/client-tools/puppetdb.conf` for Windows). For more
```

Comment:
Should this be `%USERPROFILE%\.puppetlabs\client-tools\puppetdb.conf` for Windows notation?